views/includes/page-templates/trust-individual-beneficial-owner.html

```diff
@@ -157,6 +166,10 @@ <h2 class="govuk-heading-m">What information we’ll show on the public register
         <input type="hidden" name="dateOfBirthYear" value="{{ formData.dateOfBirthYear }}"/>
       {% endif %}
 
+      <input type="hidden" name="trustCreationDateDay" value="{{ pageData.trustData.creationDateDay }}"/>
```

Comment:
Again, the 'creation' fields here look wrong given this PR is all about cease dates.

Reply:
updated now to not use these